packages/venia-concept/src/components/Input/helpers.js

```diff
@@ -0,0 +1,22 @@
+import { isString } from 'util';
+
+const emptyFieldMessage = 'This field cannot be empty';
+export const defaultIsRequired = value =>
+    isString(value) && value.length > 0 ? null : emptyFieldMessage;
+
+export const createComplexValidator = validators => (value, values = null) => {
```

Comment:
I believe in general it is better to at least move this to util/ folder. Or create services/ folder and create validator.js file there.

Reply:
https://github.com/magento-research/pwa-studio/pull/593#discussion_r238682240